apps/shooting-clubs/acceptance/features/second-contact-name.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const steps = require('../../');
+
+Feature('Second contact name step');
+
+Before((
+  I,
+  secondContactNamePage
+) => {
+  I.visitPage(secondContactNamePage, steps);
+});
+
+Scenario('The correct form element are present', (
+  I,
+  secondContactNamePage
+) => {
+  I.seeElement('#second-contact-name');
```

Comment:
Now you've added the element of to pages, this should now be secondContactNamePage['second-contact-name-id']